pkg/install/images/images.go

```diff
@@ -67,47 +72,179 @@ import (
 
 const mockNamespaceName = "mock-namespace"
 
-func ExtractAddonsFromDockerImage(ctx context.Context, log logrus.FieldLogger, dockerBinary string, imageName string) (string, error) {
-	tempDir, err := os.MkdirTemp("", "imageloader*")
+type SourceDestImages struct {
```

Comment:
This seems to be a single source-destination. Maybe we should call it something like `ImageSourceDest` (without `s`)?

Reply:
yeah that works. I wasn't too happy about the naming either.